nlp/src/main.py

```diff
@@ -77,17 +77,24 @@ async def get_language(request: TextRequest):
 
     results = []
     for text in request.text:
+        languages = []
         prediction = language_model.predict(text, k=2)
-        languages = [x.replace("__label__", "").replace("_Latn", "")
-                          for x in prediction[0]]
-        predictions = ["{:.8f}".format(y) for y in prediction[1]]
+
+        for x in prediction[0]:
+            lang_tag = x.replace("__label__", "").replace("_Latn", "")

```

Comment:
This doesn't work for alphabets other than Latin. When I use k-pop lyrics as input, it returns this:
```json
"language": [
    "kor_Hang",
    "ady_Cyrl"
],
"predicted": [
    "1.00001001",
    "0.00001000"
]
```

It would be better to just do an `rsplit` by underscore to remove the alphabet suffix.

Reply:
a more descriptive name instead of `x` would also work nice.

Reply:
Text is splitted as you suggested and value before last one should be language code.

Reply:
Tested on K-pop too 🫡